packages/amplify_core/lib/src/types/exception/storage/key_not_found_exception.dart

```diff
@@ -0,0 +1,19 @@
+// Copyright Amazon.com, Inc. or its affiliates. All Rights Reserved.
+// SPDX-License-Identifier: Apache-2.0
+
+import 'package:amplify_core/amplify_core.dart';
+
+/// {@template amplify_core.storage.key_not_found_exception}
+/// Exception thrown when the key is not found in the storage.
```

Comment:
question: In the case of s3 does this mean that the key is not in the bucket? I found word "storage" a little confusing here bc wasn't sure if it referred to local/cloud storage. Are these comments/descriptions part of cross-platform error definitions or can we change them?

If we can change it I dk if can be changed to clarify like "... not found in the storage service."

Reply:
Yea the specific message and recovery suggestion can be customized when initiate this exception according to specific occasion. storage service sounds good to me. 👍